src/main/java/slimeknights/tconstruct/plugin/craftingtweaks/TinkersCraftingGridProvider.java

```diff
@@ -0,0 +1,32 @@
+package slimeknights.tconstruct.plugin.craftingtweaks;
+
+import net.blay09.mods.craftingtweaks.api.ButtonAlignment;
+import net.blay09.mods.craftingtweaks.api.CraftingGridBuilder;
+import net.blay09.mods.craftingtweaks.api.CraftingGridProvider;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import slimeknights.tconstruct.TConstruct;
+import slimeknights.tconstruct.tables.menu.CraftingStationContainerMenu;
+
+public class TinkersCraftingGridProvider implements CraftingGridProvider {
+
+  @Override
+  public String getModId() {
+    return TConstruct.MOD_ID;
+  }
+
+  @Override
+  public boolean handles(AbstractContainerMenu menu) {
+    return menu instanceof CraftingStationContainerMenu;
```

Comment:
Doesn't `menu` have a `getType()` method we can use instead of doing an `instanceof` check? Feels more appropriate to do so, and it would be faster.

Reply:
Yes it does

Reply:
Turns out, crafting tweaks crashes if you use `getType` in this context on certain screens, guess not every menu supports that function (blame dumb mojang design). So guess the old code is required.

Reply:
Opening the inventory, might have been creative inventory specific